typegate/tests/e2e/templates/templates_test.ts

```diff
@@ -29,9 +29,10 @@ const modifiers: Record<string, (dir: string) => Promise<void> | void> = {
       );
     }
   },
+  "node": () => {},
```

Comment:
You have here the ability to replace the import to match to corresponding one in the project (e.g. we want to test on current commit, not the previous published version): see the function above for Deno. For node, you will have to edit package.json and make sure https://github.com/metatypedev/metatype/blob/main/dev/publish.ts is run to create the package.